spec/models/item_orders_spec.rb

```diff
@@ -23,5 +23,15 @@
 
       expect(item_order_1.subtotal).to eq(200)
     end
+
+    it '#fulfill' do
```

Comment:
I did not want to, but I made a test for this lil model method!

Reply:
![](https://media.giphy.com/media/Js8piewy3bzsJUfovr/giphy.gif)